VulnerableApplication/Controllers/HomeController.cs

```diff
@@ -8,6 +8,9 @@ namespace VulnerableApplication.Controllers
 {
     public class HomeController : Controller
     {
+        public static string SonarPassword = "foo";
+        public static string GitHubPassword = "bar";
```

Comment:
[CODE_SMELL] 10 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.